### PR TITLE
PKSD-4571 CLOUD-136 Enable SASL

### PR DESCRIPTION
**Pull request by k2n**
- Leverage the forked repo as the baseline to build highly available Kafka cluster. 
- Added SASL with SCRAM-SHA-256 support. 
- Inherit this from `parkside-securities/monorepo` and customize the behavior.

### CODE REVIEW

**Review by 223kazuki — APPROVED** [review]
I confirmed this image with argo.

File: kafka/30bootstrap-service.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: bootstrap
+  name: kafka-broker
   namespace: kafka
 spec:
   ports:
-  - port: 9092
+    - port: 9094
```
Comment (yijiezhen) [inline kafka/30bootstrap-service.yml]: does this matter? the new setting shifts right

File: kafka/30bootstrap-service.yml [hunk 2]
```diff
@@ -2,10 +2,10 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: bootstrap
+  name: kafka-broker
```
Comment (yijiezhen) [inline kafka/30bootstrap-service.yml:5]: change the name of the file as well?

**Review by yijiezhen — APPROVED** [review]

**Review by kgann — APPROVED** [review]

File: kafka/30bootstrap-service.yml (hunk 2 above)
Comment (k2n) [reply kafka/30bootstrap-service.yml:5]: The file name is referred in different places so I'd like to keep it as-is.

**Pull request merged**